src/www/ui/api/Middlewares/RestAuthMiddleware.php

```diff
@@ -94,6 +95,11 @@ public function __invoke($request, $response, $next)
           $tokenValid->getCode());
       }
     }
-    return $response;
+    return $response
+      ->withHeader('Access-Control-Allow-Origin', $SysConf['SYSCONFIG']['CorsOrigins'])
+      ->withHeader('Access-Control-Expose-Headers', '*')
+      ->withHeader('Access-Control-Allow-Headers', 'X-Requested-With, Content-Type, Accept, Origin, Authorization')
```

Comment:
Currently, following headers are accessed by FOSSology. Please add them to the list as well.
`action`, `active`, `copyright`, `Content-Type`, `description`, `filename`, `filesizemax`, `filesizemin`, `folderDescription`, `folderId`, `folderName`, `groupName`, `ignoreScm`, `license`, `limit`, `name`, `page`, `parent`, `parentFolder`, `public`, `reportFormat`, `searchType`, `tag`, `upload`, `uploadDescription`, `uploadId` and `uploadType`.

Reply:
Now I understand why public APIs don't generally read parameters from headers :sweat_smile:

Reply:
But @GMishx I think we don't need to add all these headers `Access-Control-Expose-Headers` this is for the response headers so in response I guess we need to add `Look-at`, `X-Total-Pages` and `Retry-After`

Reply:
@GMishx updated as you suggested and thanks for providing the headers list :)